airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts

```diff
@@ -513,15 +503,15 @@ describe("Connection sync modes", () => {
     checkSuccessResult();
 
     goToSourcePage();
-    openSourceDestinationFromGrid(sourceName);
-    openSourceDestinationFromGrid(destName);
+    openSourceOverview(sourceName);
+    openConnectionOverviewByDestinationName(destName);
 
     goToReplicationTab();
 
-    searchStream("cities");
+    searchStream(streamName);
 
-    checkCursorField("city");
-    checkPrimaryKey("city_code");
+    checkCursorField(streamName, "city");
+    checkPrimaryKey(streamName, ["city", "city_code"]);
```

Comment:
Good catch! Few points:
1. "city" should not be here
2. test is passed, and that's wrong. I've checked `checkDropdownField` function - there was a partial string check instead of an exact match. So it's fixed. Thanks!)